spec/features/users/00_authenticated_user_spec.rb

```diff
@@ -2,14 +2,31 @@
 
 RSpec.feature "as a visitor" do
   scenario "I should see a link to login" do
+    User.create(first_name: "Billy",
+             last_name: "Goat",
+             username: "billygoat",
+             password: "pass",
+             address: "123 Any St, Denver, CO 80202",
+             email: "billygoat@gmail.com"
+            )
+
     visit root_path
-    click_on "Login"
+
+    within('.navbar') do
```

Comment:
I liked that you wrapped the nav bar in a div class so it clicks the proper login button on sessions new erb.